client-coroutines/src/main/kotlin/io/provenance/client/coroutines/PbCoroutinesClient.kt

```diff
@@ -87,18 +97,15 @@ open class PbCoroutinesClient(
             )
         }
 
-    private fun mkTx(body: TxOuterClass.Tx.Builder.() -> Unit): TxOuterClass.Tx =
-        TxOuterClass.Tx.newBuilder().also(body).build()
-
     suspend fun estimateTx(baseReq: BaseReq): GasEstimate {
-        val tx = mkTx {
+        val transaction = tx {
             body = baseReq.body
             authInfo = baseReq.buildAuthInfo()
-            signaturesList.add(ByteString.EMPTY)
```

Comment:
I don't think this ever worked as this was an emptyList under the hood (not mutable)... maybe I'm wrong but it blew up for me when testing

Reply:
That could easily be true, as this was only recently changed.